BGL/include/CGAL/boost/graph/Face_filtered_graph.h

```diff
@@ -155,7 +155,13 @@ struct Face_filtered_graph
   template <typename FacePatchIndexMap, class FacePatchIndexRange>
   Face_filtered_graph(const Graph& graph,
                       const FacePatchIndexRange& selected_face_patch_indices,
-                            FacePatchIndexMap face_patch_index_map)
+                            FacePatchIndexMap face_patch_index_map
+                            #ifndef DOXYGEN_RUNNING
```

Comment:
note that the `#ifndef` is not needed as no documentation is associated to this function